src/block/header/style.scss

```diff
@@ -41,6 +41,17 @@
 	}
 }
 
+.ugb-header__buttons {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-left: -0.5em;
+	margin-right: -0.5em;
+	> * {
+		margin: 0 0.5em 0.5em;
+	}
+}
+
```

Comment:
Please place the entire code inside the `.ugb-header.ugb-header--v3` block to make it apply to v3 only